JourneysSquad/JourneysSquad/View/BillListView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+/// A view displaying the list of bills.
```

Comment:
There should be a blank line:
```suggestion
import SwiftUI

/// A view displaying the list of bills.
```

Please run linters before requesting the review.

Reply:
Fixed